client/src/leap/soledad/client/api.py

```diff
@@ -70,6 +71,26 @@
 """
 SOLEDAD_CERT = None
 
+# A whitelist of modules from where to collect plugins dynamically.
+# For the moment restricted to leap namespace, but the idea is that we can pass
+# other "trusted" modules as options to the initialization of soledad.
```

Comment:
The idea of plugins is awesome, but thinking of Soledad as a standalone module does a whitelist still make sense? I thought about maybe adding a `leap.soledad.client.registerPlugin()` function that would update the list of plugged in modules. That function would be called by `leap.mail` and `leap.keymanager` explicitelly to register their intention to plug modules in soledad, instead of the opposite. How does that sound?